index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Emmanuel and Tressor</title>
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main>
+        <header class="header">
+            <h1 id="title">Our Survey Form</h1>
+            <p id="description">
+                Thank you for taking out your time to complete the form.
+            </p>
+        </header>
+        <form id="survey-form" action="">
+            <div class="form-group">
+                <label for="name" id="name-label">Name:</label>
+                <input type="text" name="name" id="name" placeholder="Enter your name" required />
+            </div>
+            <div class="form-group">
+                <label for="email" id="email-label"> Email:</label>
+                <input type="email" name="email" id="email" class="form-control" placeholder="Enter your Email" required />
+            </div>
+
+            <div class="form-group">
+                <label for="phoneNumber" id="number-label">Age</label>
+                <input type="number" name="phoneNumber" id="number" class="form-control" placeholder="Enter Your Phone number" min="18" max="50" required />
+            </div>
+
+            <div class="form-group">
+                <label for="number" id="number-label">Age <span>(optional)</span></label
+          >
```

Comment:
. Kindly check how your **tags** are closed. Try to align them

Reply:
We can improve this review message by being more descriptive and giving a reason for the requested change.

> First state the aimed problem ❗ , politely offer a suggestion 😏 and conclude with a reason or importance of making that change.